### DISCUSSION TITLE
[Release Vote] Vote for Release V0.4.1-rc1

### DISCUSSION THREAD

**Discussion by baiyangtx** [apache/amoro#1261] [Polls]
Hi Everyone,

I propose that we release the following RC as the official Arctic 0.4.1 release.

The commit ID is https://github.com/NetEase/arctic/commit/afbd1c04e8ac93c6f2ea0c675a854c4ee52fa3e0

This corresponds to the tag: v0.4.1-rc1
https://github.com/NetEase/arctic/commits/v0.4.1-rc1
https://github.com/NetEase/arctic/tree/v0.4.1-rc1
Please check out, build, and test.

[ ] +1 Release this as Arctic 0.4.1
[ ] -1 Do not release this because...

Please vote in the next 72 hours and add your test steps when voting.

**Comment by wangtaohz** [comment]
+1

I tested it locally :

- Ran the unit test of core/ams/hive/spark/optimizer locally
- Ran optimizing integration test locally
- Installed the local ams environment, and ran the quick start guild
- Tested Optimizing of Iceberg format table and Mixed Iceberg format table

**Comment by baiyangtx** [comment]
-1 
Can't start AMS using derby
